pkg/apis/build/v1alpha1/build_pod.go

```diff
@@ -633,16 +635,25 @@ func (b *Build) rebasePod(secrets []corev1.Secret, images BuildPodImages, config
 	}, nil
 }
 
-func (b *Build) cacheVolume(os string) corev1.VolumeSource {
-	if b.Spec.CacheName != "" && os != "windows" {
-		return corev1.VolumeSource{
+func (b *Build) cacheVolume(os string) []corev1.Volume {
+	if b.Spec.CacheName == "" {
+		return []corev1.Volume{}
```

Comment:
Can we return the this nonexistent cacheVolume on windows instead of the empty dir?

Reply:
I don't know why an `emptyDir` is used for Windows until now, so I didn't want to change that.
Also, I don't have access to a Windows machine to test it. But I can update the PR, if you think it's safe.

Reply:
We don't (hopefully yet) support persistent volume claims on windows so similar to linux builds we just used an emptyDir. However, with this PR on linux we just don't mount a cache dir if no cache is provided so it makes sense to just apply a similar strategy on windows.

Reply:
We updated the PR: We now use the same strategy for Windows (always) as for Linux when `CacheName` is empty.
As I mentioned before, I can not test this as I don't have an Windows Environment.

Reply:
Awesome! We can verify this on windows @phil9909.